src/scss/components/_alert.scss

```diff
@@ -1,44 +1,30 @@
 @use 'microscope-sass/lib/bem';
 
-@import '../mixins/prefix';
-@import '../settings';
-
-.#{prefix('alert')} {
-  @include margin(true, $properties: padding);
-  @include margin('auto', $properties: margin-bottom);
-  display: flex;
-  justify-content: flex-start;
-
-  @include bem.modifier('info') {
-    background-color: var(--of-alert-info-bg);
-  }
-
-  @include bem.modifier('error') {
-    background-color: var(--of-alert-error-bg);
-
-    .fa-icon {
-      color: var(--of-color-danger);
+// Temporary CSS override to get the colour of the icon to work
+.openforms-theme {
+  .utrecht-alert {
+    @include bem.modifier('error') {
+      .utrecht-alert__icon {
+        color: var(--utrecht-alert-icon-error-color);
```

Comment:
this is not an existing design token right? I'd namespace it under our own design tokens then: `of.utrecht-alert.icon.error.color`.

Reply:
This is an existing token! But it doesn't work yet